tutorials/fundamentals/regex-no-start.md

```diff
@@ -0,0 +1,15 @@
+To check if a string does not start with specific characters using a regular expression, use the `test()` function and negate it.
+Make sure to also use a `^` character as this signals to only check the start of the string.
```

Comment:
Make sure your regular expression starts with `^`, which is a special character that represents the start of the string.